nx-X11/include/extensions/Imakefile

```diff
@@ -42,15 +42,6 @@ XINERAMAHEADERS = Xinerama.h panoramiXext.h panoramiXproto.h
 #if BuildRender || BuildXftLibrary
 RENDERHEADERS = render.h renderproto.h
 #endif
-#if BuildXfixes || BuildXfixesLibrary
-XFIXESHEADERS = xfixeswire.h xfixesproto.h
-#endif
-#if BuildDamage || BuildDamageLibrary
-DAMAGEHEADERS = damagewire.h damageproto.h
-#endif
-#if BuildComposite || BuildCompositeLibrary
-COMPOSITEHEADERS = composite.h compositeproto.h
-#endif
```

Comment:
Maybe we should actually look at how it could be possible to squeeze nx-X11/programs/Xserver/hw/nxagent into a real hardware driver project building against xserver-xorg.

However, this will fail, as we will need to continue building against libnx-x11-6, libnx-xrender3, libnx-xinerama1, etc.